developer_copr_setup.md

```diff
@@ -0,0 +1,157 @@
+## Developer Copr Setup:
+
+### Background
+
+ManageIQ requires some upstream rpms for use on our CentOS appliances.
+Copr makes these rpms publicly available for the community build.
+
+These steps will walk you through:
+
+* Create a fedora / copr account
+* Create a copr project
+* add repositories for dependencies
+* specify the chroot
+* add any additional packages to the minimal chroot
+* specify the `src.rpm` to build
+* host existing source rpm (srpm) publicly
+* mock, test, and build the rpm
+* incorporate the copr yum repo into our upstream community build.
+
+
+###  Create a fedora account (used as a copr account)
+* Create a fedora account: http://copr.fedoraproject.org/
+* Review [copr tutorial](https://fedorahosted.org/copr/wiki/ScreenshotsTutorial).
+
+### Create a copr project
+
+* The project groups similar rpms together.
+* Put rpms together based upon a common build environment and package requirements.
+* Also, group rpms based upon how the yum repository will be consumed.
+* Name the project correctly since you can't rename it.
+
+### Configure the project's settings:
+
+* [chroot](https://help.ubuntu.com/community/BasicChroot):
+  * **epel-x86_64**
+* Edit this chroot and "specify additional packages to be always present in minimal buildroot":
+  * If you need to build a ruby gem that utilizes the SCL, you need the following:
+    * **scl-utils-build ruby193-build**
```

Comment:
Maybe, we'll need to fix this if we build anything that requires ruby.  I"m not even sure the exact SCL prefix name.  I guess it's ruby22 but I don't know.